README.md

```diff
@@ -21,6 +21,12 @@ See [`CONTRIBUTING.md`](./CONTRIBUTING.md)
 $ yarn
 ```
 
+### build (if first time building dev environment, must be run before yarn dev)
+
+```sh
+$ yarn build
+```
+
```

Comment:
This is a bit unrelated to the overlay issue, but added to the README that upon first run, must run `yarn build` prior to `yarn dev`.